ui/pages/signup/index.js

```diff
@@ -172,14 +174,16 @@ export default function Signup() {
                   setError('')
                 }}
                 className={`block w-full min-w-0 rounded-l-lg px-3 py-2 shadow-sm focus:border-blue-500 focus:ring-blue-500 sm:text-sm ${
-                  errors.domain ? 'border-red-500' : 'border-gray-300'
+                  errors['org.subdomain'] ? 'border-red-500' : 'border-gray-300'
                 }`}
               />
               <span className='inline-flex select-none items-center rounded-r-lg border border-l-0 border-gray-300 bg-gray-50 px-3 text-2xs text-gray-500 shadow-sm'>
                 .{baseDomain}
               </span>
             </div>
-            {errors.domain && <ErrorMessage message={errors.domain} />}
+            {errors['org.subdomain'] && (
```

Comment:
No need for using string keys here. Also, is it `errors?.org?.subDomain`? https://github.com/infrahq/infra/blob/4178903220e8a262b20523fb6aa529b5650f7f0f/api/signup.go#L18

```suggestion
            {errors?.org?.subdomain && (
```

Reply:
this will not work - i tried and it always returns `undefined` as the value